libredox/src/to_num.rs

```diff
@@ -4,14 +4,14 @@ use get_slice::GetSlice;
 
 /// Parse the string to a integer using a given radix
 pub trait ToNum {
-    fn to_num_radix(&self, radix: usize) -> usize;
-    fn to_num_radix_signed(&self, radix: usize) -> isize;
-    fn to_num(&self) -> usize;
-    fn to_num_signed(&self) -> isize;
+    fn to_num_radix(&self, radix: u32) -> u32;
```

Comment:
Perhaps to_num<T> makes more sense than forcing u32 here? It would limit parsing to numbers below 4 billion